heroku/cli
# migrate(W-14169233): spaces: Upgrade drains:set

This PR moves `drains:set` to the CLI and migrates to TypeScript

### Test Instructions

1. checkout this branch and run `yarn && yarn build && cd/packages/cli`
2. run `./bin/run drains:set DRAIN_URL --space YOUR_SPACE`
3. Verify the output is the same as the original
